### DISCUSSION TITLE
How to have different name for output .html file

### DISCUSSION THREAD

**Discussion by vnpradhan** [badeball/cypress-cucumber-preprocessor#1042] [Q&A]
The HTML output file name is generated with name - index.html
How to change the name of this file while generation?

**Comment by badeball** [comment]
https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/html-report.md

```json
{
  "html": {
    "enabled": true,
    "output": "foobar.html"
  }
}
```
